relay.go

```diff
@@ -160,24 +246,26 @@ func (r *Relayer) handleNonCallReq(f *Frame) error {
 		items = r.inbound
 	}
 
-	r.RLock()
-	item, ok := items[f.Header.ID]
-	r.RUnlock()
+	item, ok := items.Get(f.Header.ID)
 	if !ok {
 		return errors.New("non-callReq for inactive ID")
 	}
+	if item.tomb {
+		// Call timed out, ignore this frame.
```

Comment:
(TODO: we should track the number of response frames we receive after the item has timed out)